src/main/java/com/ldtteam/structurize/blocks/interfaces/IBlueprintDataProvider.java

```diff
@@ -24,6 +24,8 @@
     public final static String TAG_TAG_NAME_LIST  = "tagNameList";
```

Comment:
Rename this interface to make it clear that it is supposed to be implemented on the block entity of the anchor.

Reply:
Possibly move to different package.